plugins/mac-videotoolbox/encoder.c

```diff
@@ -1346,6 +1366,7 @@ static void vt_defaults(obs_data_t *settings, void *data)
 				    type_data->codec_type == kCMVideoCodecType_H264 ? "high" : "main");
 	obs_data_set_default_int(settings, "codec_type", kCMVideoCodecType_AppleProRes422);
 	obs_data_set_default_bool(settings, "bframes", true);
+	obs_data_set_default_bool(settings, "spatial_aq", false);
```

Comment:
Shouldn't this be `true` to ensure that the default value is used by default and we don't introduce non-default behaviour instead?

Reply:
Yeah just force of habit of setting a new key to false by default.